app/assets/stylesheets/_bootswatch.scss

```diff
@@ -0,0 +1,149 @@
+// Journal 3.3.5
+// Bootswatch
+// -----------------------------------------------------
+
+$web-font-path: "https://fonts.googleapis.com/css?family=News+Cycle:400,700" !default;
+
+@mixin web-font($path){
+  @import url("#{$path}");
+}
+@include web-font($web-font-path);
+
+// Navbar =====================================================================
+
+.navbar {
+  font-size: 18px;
+  font-family: $font-family-sans-serif;
+  font-weight: $headings-font-weight;
+
+  &-default {
+
+    .badge {
+      background-color: #000;
+      color: #fff;
+    }
+  }
+
+  &-inverse {
+
+    .badge {
+      background-color: #fff;
```

Comment:
Color literals like `#fff` should only be used in variable declarations; they should be referred to via variable everywhere else.